client/next.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  env: {
+    GOOGLE_MAPS_API_KEY: "AIzaSyDzA6_0rS4FctjvVkN3yyCOypdVoaz5Z9s"

```

Comment:
[boulder] You are trying to have someone steal my key and run my bill to $100000000000000

Reply:
Should be fixed now, found out how to make the .env.local work, it was a dumb mistake on my part